post.hbs

```diff
@@ -64,32 +64,24 @@
             </section> }}
             </footer>
         </article>
-
-        <aside class="gh-readmore">
-            <div class="gh-readmore-inner w-full max-w-screen-xl mx-auto my-0">
-                <div class="gh-readmore-next">
-                    {{#next_post}}
-                        <a href="{{url}}">
-                            <i class="fa fa-chevron-left" aria-hidden="true"></i>
-                            &nbsp;
-                            <h4>
-                                {{title}}
-                            </h4>
-                        </a>
-                    {{/next_post}}
-                </div>
-                <div class="gh-readmore-prev">
-                    {{#prev_post}}
-                        <a href="{{url}}">
-                            <h4>
-                                {{title}}
-                            </h4>
-                            &nbsp;
-                            <i class="fa fa-chevron-right" aria-hidden="true"></i>
-                        </a>
-                    {{/prev_post}}
-                </div>
-            </div>
+        
+        {{!-- implementation of related posts --}}
+        <article>
+            <br> {{!-- spacing --}}
+            <hr> {{!-- divider --}}
+            <br> 
+            <h2>
+                Related Articles
+            </h2> {{!-- title --}}
+            <br>
+            <div id="related-articles" class="grid grid-cols-1 lg:grid-cols-3 gap-x-20 gap-y-10 m-auto">
+                {{#get "posts" include="tags" limit="3" filter="tags:{{primary_tag.slug}}+id:-{{id}}" order="title asc" as |related|}}
```

Comment:
This is a good start but not very useful for our usecases at the moment. Could you use an OR filter for all tags (see https://ghost.org/docs/content-api/#filtering for more details on the NQL syntax)? Also see instructions at https://forum.ghost.org/t/show-related-posts-from-all-associated-tags/14005.

Reply:
Fixed